cmake/Platform/Common/MSVC/Configurations_msvc.cmake

```diff
@@ -85,6 +85,9 @@ ly_append_configurations_options(
         /Ot             # Favor fast code over small code
         /Oi             # Use Intrinsic Functions
         /Oy             # Omit the frame pointer
+        /Zi
+        /MD
+        /DEBUG
```

Comment:
The changes to this file should be reset to the previous commit.
This is adding debug information to the MSVC builds